tests/test_notify_v2.py

```diff
@@ -139,38 +141,45 @@ def test_notify_or_queue(self):
         with self.subTest("Should attempt to notify immediately"):
             with mock.patch("dss.events.handlers.notify_v2.notify") as mock_notify:
                 with mock.patch.object(SQSMessenger, "send") as mock_send:
-                    md = dict(**metadata_document, **dict(event_type="CREATE"))
-                    key = f"bundles/{uuid4()}.{datetime_to_version_format(datetime.datetime.now())}"
-                    notify_or_queue(replica, subscription, md, key)
-                    mock_notify.assert_called()
-                    mock_send.assert_not_called()
-                    keys = [a[0][2] for a in mock_notify.call_args_list]
-                    self.assertIn(key, keys)
+                    with mock.patch("dss.events.handlers.notify_v2.update_subscription_stats") as mock_update:
+                        mock_update.return_value = None
+                        md = dict(**metadata_document, **dict(event_type="CREATE"))
```

Comment:
Out of curiosity, is there a reason this isn't written

```suggestion
                        md = dict(**metadata_document, event_type="CREATE")
```

?

Reply:
im not sure, @xbrianh wrote that section, i tested it with your suggestion and it was able to pass.